examples/player/simid_player.js

```diff
@@ -393,6 +417,23 @@ class SimidPlayer {
         .then(() => this.destroyIframeAndResumeContent_());
   }
 
+  /**
+   * Skips the ad and destroys the ad iframe.
+   * @param {StopCode} reason The reason the ad will stop.
+   */
+  skipAd(reason = StopCode.PLAYER_INITATED) {
+    // The iframe is only hidden on ad stoppage. The ad might still request
+    // tracking pixels before it is cleaned up.
+    this.hideSimidIFrame_();
+    const closeMessage = {
+      'code': reason,
+    }
```

Comment:
This variable goes unused (I don't think you need it, skipAd doesn't take in a code the way a stopAd message does)
https://interactiveadvertisingbureau.github.io/SIMID/#simid-player-adSkipped

Reply:
yeah I was a little confused that I didn't see SkipCodes in simid protocol. Got rid of this variable